lib/rucio/web/ui/flask/bp.py

```diff
@@ -142,6 +142,18 @@ def view_maker(template, title):
     return lambda: authenticate(template=template, title=title)
 
 
+class AuthenticatedBlueprint(Blueprint):
+    """
+    Enforce authorisation of blueprints by default
+    returns a normal blueprint otherwise
+    """
+
+    def __init__(self, enforce_auth: bool = True, *args, **kwargs) -> None:
```

Comment:
I think the `enforce_auth` option should be taken out. When the AuthenticatedBlueprint is used, it always must enforce auth. Otherwise the standard blueprint should be used.